### DISCUSSION TITLE
Password Expiry Health Status - sorting by Alert

### DISCUSSION THREAD

**Discussion by plearn2023** [vmware/powershell-module-for-vmware-cloud-foundation-reporting#116] [Community Help]
Just thinking.
Would it makes sense to sort using "Status"
e.g. out of 10 accounts only one is expiring in next 7 days? 
It should be top?

**Comment by tenthirtyam** [comment]
We sort by component, resource. You can run the `-failuresOnly` switch to show only the critical items.

**Reply by tenthirtyam** [reply]
Please note that in v2.0.0 this will also pull straight out of the SoS Health Summary response JSON (via API) and will be much faster.

**Reply by plearn2023** [reply]
Thanks a lot. I cannot agree more... 🖖
